diagnostic.md

```diff
@@ -2,37 +2,43 @@
 
 What are three advantages of using Ember?
 
-```sh
-# your answer here
+```md
+Dynamic Data Binding
+Client Side Casching
+URLs!
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
-```sh
-# your answer here
+```md
+They use handlebars within their view framework. That is to say, each view has its own template which may be compiled along with many componants which in turn may have their own templates. These templates are all pumped into a basic ember html boilerplate and may also have extensions with in their HB templates.
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
-```sh
-# your answer here
+```md
+View layer
+Routes layer
```

Comment:
Correct! Here's some more for good measure 

Router
Routes
Models
Templates
Components
Adapters
Controllers (1.0)
Views(1.0)